docs/blogs-and-videos.md

```diff
@@ -0,0 +1,16 @@
+---
+id: blogs-and-videos
+title: Blogs & Videos
+---
+
+Here are some links to other blog posts and videos which may provide further examples and reading.
+
+### From the Expedia Group team
+
+#### graphql-kotlin
+* 📝 [graphql-kotlin: Generate a GraphQL schema from Kotlin code](https://medium.com/expedia-group-tech/graphql-kotlin-generate-a-graphql-schema-from-kotlin-code-21d1dc2f6e27) - First intial release and open source annoucement
+* 📝 [Release of graphql-kotlin 1.0.0!](https://medium.com/expedia-group-tech/release-of-graphql-kotlin-1-0-0-791ad85d3116) - Full release announcement
+
+#### GraphQL
+* 📺 [Creating a federated schema for a global company](https://youtu.be/MuD3TAP0D9Y)
```

Comment:
maybe we should specify the conference/summit/meetup as well?

Reply:
We could but I am not sure if that is relevant to the content in the videos. I think we can have this page just be simple links for now